internal/terraform/transform_diff.go

```diff
@@ -24,6 +24,13 @@ type DiffTransformer struct {
 	Config   *configs.Config
 }
 
+// GraphNodeAttachProvider is an interface that must be implemented by nodes
```

Comment:
comment needs updating :)

Reply:
I removed the interface instead and replaced it with a check for the only type that implements it. It will probably stay the only type implementing this so I felt this was a bit clearer.